plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt

```diff
@@ -23,16 +22,6 @@ open class KtlintBasePlugin : Plugin<Project> {
             target.tasks.withType(BaseKtlintCheckTask::class.java).configureEach(it)
         }
 
-        target.tasks.withType(BaseKtlintCheckTask::class.java).configureEach {
-            val objects = target.objects
-            val gradleVersion = GradleVersion.version(target.gradle.gradleVersion)
-            if (gradleVersion < GradleVersion.version("6.0")) {
-                it.runner = objects.newInstance(JavaExecKtLintRunner::class.java, target)
-            } else {
-                it.runner = objects.newInstance(WorkerApiKtLintRunner::class.java)
-            }
-        }
-
```

Comment:
What happened to this?

Reply:
`JavaExecKtLintRunner` was used when plugin was running with Gradle prior to 6.0 version. As minimal supported Gradle version was bumped to `6.5.1` - I've removed it.

Reply:
Ah, moved up here:

https://github.com/JLLeitschuh/ktlint-gradle/pull/420/files#diff-16dd0c2785c1b92755de7e2eaef5f722a769733f7917e1dda9995c5d60f17123R159

Makes sense to me! Thanks!